Packs/Base/Scripts/ValidateContent/ValidateContent.py

```diff
@@ -325,14 +346,13 @@ def main():
         result = validate_content(filename, file_contents, content_tmp_dir.name)
         outputs = []
         for validation in result:
-            for item in validation.values():
-                for error in item.get('outputs', []):
-                    if error.get('ui') or item.get('file-type') in {'py', 'ps1'}:
-                        outputs.append({
-                            'Name': item.get('display-name'),
-                            'Error': error.get('message'),
-                            'Line': error.get('line-number'),
-                        })
+            adjust_linter_row_and_col(validation)
+            if validation.get('ui') or validation.get('fileType') in {'py', 'ps1'}:
```

Comment:
i suppose the 2 loops that were here are no longer required cause of the changes in the sdk?

Reply:
yea exactly